autogluon/inference/docker/0.4.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,35 @@
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference:1.9.0-cpu-py38-ubuntu20.04-sagemaker
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON_VERSION=3.8.10
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG TORCH_VER=1.10.0+cpu
+ARG TORCH_VISION_VER=0.11.0+cpu
+ARG NUMPY_VER=1.21.5
+ARG AUTOGLUON_VERSION=0.4.0
+
+RUN pip --no-cache-dir install --upgrade --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip --no-cache-dir install --upgrade wheel setuptools \
+ && pip uninstall -y dataclasses \
+ && pip install --no-cache-dir -U torch==${TORCH_VER} torchvision==${TORCH_VISION_VER} -f https://download.pytorch.org/whl/cpu/torch_stable.html \
+    numpy==${NUMPY_VER} \
+    autogluon==${AUTOGLUON_VERSION}
+
+EXPOSE 8080 8081
```

Comment:
The AG `THIRD-PARTY-LICENSES.txt` that is downloaded into the training images seems to be missing in this dockerfile. Is this intentional?

Reply:
added